pkg/controller/pxc/tls.go

```diff
@@ -56,15 +56,49 @@ func (r *ReconcilePerconaXtraDBCluster) createSSLByCertManager(cr *api.PerconaXt
 	}
 	ownerReferences := []metav1.OwnerReference{owner}
 
-	issuerName := cr.Name + "-pxc-ca"
+	issuerName := cr.Name + "-pxc-issuer"
+	caIssuerName := cr.Name + "-pxc-ca-issuer"
 	issuerKind := "Issuer"
 	issuerGroup := ""
 	if cr.Spec.TLS != nil && cr.Spec.TLS.IssuerConf != nil {
 		issuerKind = cr.Spec.TLS.IssuerConf.Kind
 		issuerName = cr.Spec.TLS.IssuerConf.Name
 		issuerGroup = cr.Spec.TLS.IssuerConf.Group
 	} else {
-		if err := r.createIssuer(ownerReferences, cr.Namespace, issuerName); err != nil {
+		if err := r.createIssuer(ownerReferences, cr.Namespace, caIssuerName, ""); err != nil {
+			return err
+		}
+
+		caCert := &cm.Certificate{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:            cr.Name + "-ca-cert",
+				Namespace:       cr.Namespace,
+				OwnerReferences: ownerReferences,
+			},
+			Spec: cm.CertificateSpec{
+				SecretName: cr.Name + "-ca-cert",
+				CommonName: cr.Name + "-ca",
+				IsCA:       true,
+				IssuerRef: cmmeta.ObjectReference{
+					Name:  caIssuerName,
+					Kind:  issuerKind,
+					Group: issuerGroup,
+				},
+				Duration:    &metav1.Duration{Duration: 87600 * time.Hour},
+				RenewBefore: &metav1.Duration{Duration: 730 * time.Hour},
```

Comment:
I think renewal every 30 days can lead to issues every 30 days.
```suggestion
				RenewBefore: &metav1.Duration{Duration: 78840 * time.Hour},
```

Reply:
@delgod that's 30 days before the expiration (in 10 years) https://cert-manager.io/docs/usage/certificate/#renewal

Reply:
In doc we need to add information that in 10 years client will have the issue with CA certificate and the maintenance should be scheduled to renew it :)

Reply:
@hors I believe we should just explain in the docs why exactly CA renewal leads to the issues. But of course, the default 10 year mark should be mentioned, too :)